presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/TableScanMatcher.java

```diff
@@ -64,35 +63,8 @@ public MatchResult detailMatches(PlanNode node, PlanNodeCost cost, Session sessi
         String actualTableName = tableMetadata.getTable().getTableName();
         return new MatchResult(
                 expectedTableName.equalsIgnoreCase(actualTableName) &&
-                        domainMatches(tableScanNode, session, metadata));
-    }
-
-    private boolean domainMatches(TableScanNode tableScanNode, Session session, Metadata metadata)
-    {
-        if (!expectedConstraint.isPresent()) {
```

Comment:
Why you decided to extract this outside of `domainMatches` even though in both cases where you use it, you check it anyway?